.github/workflows/build.yaml

```diff
@@ -90,3 +99,26 @@ jobs:
       run: |
         cargo install --path .
         agora --version
+
+    - name: Package
+      id: package
+      if: startsWith(github.ref, 'refs/tags/')
+      env:
+        TARGET: ${{ matrix.target }}
+        REF: ${{ github.ref }}
+        OS: ${{ matrix.os }}
+      run: ./bin/package
+
+    - name: Pre-release Check
+      id: prerelease
+      run: cargo run -p prerelease -- --reference ${{ github.ref }}
+
+    - name: Publish
+      uses: softprops/action-gh-release@v0.1.5
+      if: startsWith(github.ref, 'refs/tags/')
+      with:
+        draft: false
+        files: ${{ steps.package.outputs.archive }}
+        prerelease: ${{ steps.prerelease.outputs.value }}
```

Comment:
Should `value` be called something like `is_prerelease`?

Reply:
Changed step to `is_prerelease`